src/hotspot/share/utilities/ostream.hpp

```diff
@@ -215,6 +215,10 @@ class stringStream : public outputStream {
   // Return number of characters written into buffer, excluding terminating zero and
   // subject to truncation in static buffer mode.
   size_t      size() const { return _written; }
+  // Borrow the currently allocated string.
+  // A stringStream may free the pointer returned
+  // if any of its methods are called, therefore
+  // take care in using this method.
```

Comment:
Comment is a bit misleading, what does "borrow" mean? 

Suggestion: "Returns internal buffer. Returned buffer is only guaranteed to be valid as long as stream is not modified"